compat/test/browser/svg.test.js

```diff
@@ -66,4 +67,15 @@ describe('svg', () => {
 			)
 		);
 	});
+
+	it('should render SVG to DOM without name (clipPathUnits) attribute manipulation', () => {
+		let svg = (
+			<svg>
+				<clipPath id="hexagon" clipPathUnits="objectBoundingBox">
+					<polygon points="0.5 0, 1 0.25, 1 0.75, 0.5 1, 0 0.75, 0 0.25" />
+				</clipPath>
+			</svg>
+		);
+		expect(svg).to.eql(svg);
```

Comment:
This test is incorrect this asserts that running the same function twice results in the same output

Reply:
@JoviDeCroock is right. The test is missing a `render` call. Right now it does this:

```jsx
const foo = {};
expect(foo).to.eql(foo);
```

You probably forgot to push somme of your commits 🙂

Reply:
like so: 

```it('should render SVG to DOM without name (clipPathUnits) attribute manipulation', () => {
		React.render(
			<svg>
				<clipPath id="hexagon" clipPathUnits="objectBoundingBox">
					<polygon points="0.5 0, 1 0.25, 1 0.75, 0.5 1, 0 0.75, 0 0.25" />
				</clipPath>
			</svg>,
			scratch
		);
		expect(serializeHtml(scratch)).to.eql(
			sortAttributes(
				'<svg><clipPath id="hexagon" clipPathUnits="objectBoundingBox"><polygon points="0.5 0, 1 0.25, 1 0.75, 0.5 1, 0 0.75, 0 0.25" /></clipPath></svg>'
			)
		);
	});